migtests/tests/pg/omnibus/expected_files/expectedAssessmentReport.json

```diff
@@ -655,9 +655,9 @@
 		{
```

Comment:
there are a couple of other schemas as well `mgi` and `rna` that are not in GH, so please verify those as well once